news/2025/2025-03-25-the-followpoint-special-behind-the-banhammer.md

```diff
@@ -0,0 +1,292 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Are there any specific misconceptions about the GMT's responsibilities or authority that you think players often misunderstand and you'd like to clarify?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>
+
+Once in a while, you can come across the statements that the Global Moderation Team is responsible for handling account restrictions, silence appeals, or account-related inquiries in general. As a matter of fact though, this belongs to the [account support team](/wiki/People/Account_support_team) set of tasks.
+
+If you are facing any issues with your account, please consult the article above and direct your concerns to one of the outlined inboxes.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+And lastly, what is your favorite GMT out of context quote?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7944724" style="background-image: url('/wiki/shared/avatars/0x84f.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/7944724">0x84f</a></p>
+
+"Hey, the."
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>
+
+"Hey, the."
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>
+
+"Hey, the."
+
+</div>
+
+---
+
+I'd like to thank all Global Moderation Team members for everything they do for osu! and their efforts to make chats a nicer place! I know that sometimes it doesn't seem like it, but they are really trying their best, so please don't let some bad apples from the past make you think all GMTs are like this, I know some of them myself and they are amazing people. <3
```

Comment:
is this referring to the people you just interviewed ("like this") or some random event months/years ago? either way this sounds passive-aggressive and confusing

```suggestion
I'd like to thank all Global Moderation Team members for everything they do for osu! and their efforts to make chats a nicer place! I know that sometimes it doesn't seem like it, but they are really trying their best. I know some of them myself and they are amazing people. <3
```